_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+
+### Scope
+
+The mission of pyOpenSci’s open peer review process is to:
+
+1. Support improving the quality, usability and discoverability of maintained scientific Python software in support of open science.
+
+2. We also support maintainers in navigating the Python packaging ecosystem.
+
+pyOpenSci reviews packages that support open reproducible science, data processing and the various stages of managing the data lifecycle. We review packages with the goal of improving package quality and usability for scientists. As such, we review packages across a spectrum of small to large user bases. The popularity of your package is not a consideration in our review process!
+
+We welcome young packages that are just entering the scientific Python ecosystem to apply for review if they are relevant to the science community and fit into at least one scope category below. We also welcome mature packages with a growing or established community!
+
+The following categories are in-scope for the pyOpenSci review process:
+* **Data retrieval:** refers to packages for accessing and downloading data from online sources, including wrappers for accessing APIs.
+* **Data extraction:** these packages aid in retrieving data from unstructured sources such as text, images, and PDFs. They might also parse scientific data types and outputs from scientific equipment.
+* **Data processing/munging:** this category focuses on tools for handling data in specific formats that scientists may be interested in working with.
+* **Data deposition:** includes tools for depositing data into scientific research repositories.
+* **Data validation and testing:** refers to tools that enable automated validation and checking of data quality and completeness that in turn support scientific workflows.
+* **Data visualization:** these are packages that enhance a scientist’s experience in visualizing and analyzing data. If your package falls into the category of data visualization, please fill out a [pre-submission inquiry](https://github.com/pyOpenSci/software-submission/issues/new?assignees=&labels=0%2Fpresubmission&template=presubmission-inquiry.md&title=) first.
+* **Workflow automation:** these tools automate and link together workflows, and as such support reproducible workflows. These tools may include build systems and tools to manage continuous integration, as well as support version control.
+* **Citation management and bibliometrics:** these are packages that facilitate managing references.
+* **Scientific software wrappers:** these tools provide a Python interface for existing scientific packages written in other languages.
+* **Database interoperability:** these packages deal with bindings and wrappers for database APIs.
+
+Similar to the previous section, you can indicate which categories are applicable to your package in one of two ways:
```

Comment:
This is nice - turn it into a breakout using a .notice block given it's supporting information.